vespalib/src/vespa/vespalib/util/alloc.cpp

```diff
@@ -328,8 +331,8 @@ MMapAllocator::salloc(size_t sz, void * wantedAddress)
         }
         buf = mmap(wantedAddress, sz, prot, flags | _G_HugeFlags, -1, 0);
         if (buf == MAP_FAILED) {
-            if ( ! _G_hasHugePageFailureJustHappened ) {
-                _G_hasHugePageFailureJustHappened = true;
+            if ( ! load_relaxed(_G_hasHugePageFailureJustHappened)) {
```

Comment:
if ( ! _G_hasHugePageFailureJustHappened.exchange(true)) {  atomic load/and store

Reply:
Is it worth a `cmpxchg` to skip a debug log?

Reply:
No, good point.